swift/src/Ice/Initialize.swift

```diff
@@ -162,6 +162,22 @@ private func initializeImpl(
 
         precondition(initData.logger != nil && initData.properties != nil)
 
+        if let sliceLoader = initData.sliceLoader {
+            initData.sliceLoader = CompositeSliceLoader(sliceLoader, DefaultSliceLoader())
+        } else {
+            initData.sliceLoader = DefaultSliceLoader()
+        }
```

Comment:
Maybe using map and nil coalescing is more "Swifty"

```
initData.sliceLoader = initData.sliceLoader.map {
    CompositeSliceLoader($0, DefaultSliceLoader()) } ?? DefaultSliceLoader()
```

Reply:
I find my version more readable, if not as Swifty.